spambayes/spambayes/port.py

```diff
@@ -1,10 +1,3 @@
-try:
-    # New in Python 2.5
-    from hashlib import md5
```

Comment:
I'm only looking at a small window, but doesn't md5 need to be imported from hashlib?

Reply:
Ah. Yes, but there's no reason to do that in the portability layer since it'll always be in hashlib (on python 2.5+) and never the original md5 module. I see that I did the imports everywhere from the md5 module instead of the hashlib module. I'll fix that with a follow-up commit.